cli/azd/cmd/provision.go

```diff
@@ -40,6 +43,10 @@ func (i *provisionFlags) bindNonCommon(local *pflag.FlagSet, global *internal.Gl
 
 func (i *provisionFlags) bindCommon(local *pflag.FlagSet, global *internal.GlobalCommandOptions) {
 	local.BoolVar(&i.preview, "preview", false, "Preview changes to Azure resources.")
+	if i.alphaFeatureManager.IsEnabled(alpha.BicepADS) {
+		local.BoolVar(&i.ignoreBicepAds, "ignore-ads", false, "Ignore Azure Deployment State (bicep only).")
```

Comment:
When I first saw this I read `ads` like `advertisements` - I thought you were going to start monetizing for us 😂